book/configuration.md

```diff
@@ -51,6 +51,15 @@ These are some important variables to look at for Nushell-specific settings:
 - `PROMPT_INDICATOR_VI_NORMAL = "〉 "`
 - `PROMPT_MULTILINE_INDICATOR = "::: "`
 
+### Configurations with built-in commands
+With new release of Nushell, we have introduced two new commands(`config nu` and `config env`) which help you quickly edit nu configurations with your preferred text editor/IDE
```

Comment:
I was thinking about the same , but kind of hesitated because I did not see other docs mention the version. Are we good to specify version number, I think this might be a general question

Reply:
"With new release of Nushell" is not clear enough and delivers less information than a specified version. Any thoughts? @fdncred

Reply:
I'd just say, starting with release v0.64 of nushell, we have introduced....